app/src/main/java/org/mozilla/tv/firefox/fxa/FxaRepo.kt

```diff
@@ -73,31 +76,34 @@ class FxaRepo(
         return accountManager.beginAuthenticationAsync()
     }
 
-    private inner class FirefoxAccountObserver : AccountObserver {
+    @VisibleForTesting(otherwise = VisibleForTesting.NONE)
+    inner class FirefoxAccountObserver : AccountObserver {
+        /**
+         * The account profile is fetched asynchronously.
```

Comment:
Perhaps this comment should be on the enum kdoc instead and we should reference it here. It can probably also we an inline comment rather than function kdoc.